examples/customize/llms/custom_llm.py

```diff
@@ -1,21 +1,32 @@
 import random
 import string
-from typing import Any
+from typing import Any, Optional
 
 from neo4j_graphrag.llm import LLMInterface, LLMResponse
+from neo4j_graphrag.llm.types import LLMMessage
 
 
 class CustomLLM(LLMInterface):
     def __init__(self, model_name: str, **kwargs: Any):
         super().__init__(model_name, **kwargs)
 
-    def invoke(self, input: str) -> LLMResponse:
+    def invoke(
```

Comment:
The `system_instruction` param is missing in the init I think?

Reply:
[a15a514](https://github.com/neo4j/neo4j-graphrag-python/pull/234/commits/a15a514b8c28e887a8d80cac69556872a6c73f9e)